worker/echo/main.go

```diff
@@ -54,6 +62,7 @@ func main() {
 
 	flag.StringVar(&logLevel, "log-level", "error", "set log level")
 	flag.BoolVar(&remoteContent, "remote-content", false, "connect as a remote content worker")
+	flag.IntVar(&sleepTime, "sleep", 0, "sleep time in seconds before echoing the response")
```

Comment:
The `flag` package has a [`DurationVar`](https://pkg.go.dev/flag#DurationVar) function that accepts a string that can be parsed to a `time.Duration`. I recommend using this over a plain integer flag. This allows input such as `-sleep 1m` to be valid input.

Reply:
This is interesting